File: iniciante/javascript/1015.js
var input = require('fs').readFileSync('stdin', 'utf8');
var lines = input.split('\n');

function distanciaEntreDoisPontos(a, b, c, d){
    return Math.sqrt(Math.pow((c - a), 2) + Math.pow((d - b), 2))
}

let entrada1 = lines.shift().split(' ')
let entrada2 = lines.shift().split(' ')

console.log(`${distanciaEntreDoisPontos(parseFloat(entrada1[0]), parseFloat(entrada1[1]), parseFloat(entrada2[0]), parseFloat(entrada2[1])).toFixed(4)}`)

File: iniciante/python/1015.py
# -*- coding: utf-8 -*-

entrada1 = input().split(' ')
entrada2 = input().split(' ')

x1 = float(entrada1[0]) 
x2 = float(entrada2[0]) 
y1 = float(entrada1[1]) 
y2 = float(entrada2[1]) 

distancia_pontos = ((x2 - x1) ** 2 + (y2 - y1) ** 2) ** (1/2)

print(f'{distancia_pontos:.4f}')
